cat/CaT-service.yaml

```diff
@@ -554,27 +554,25 @@ paths:
             - buyer
       summary: Create Project from an Agreement (SCC-440)
       
-  /tenders/projects/{proc-id}/confirm:
+  /tenders/projects/{proc-id}/name:
     parameters:
       - $ref: '#/components/parameters/ProcurementParam'
-    post:
-      deprecated: true
-      description: Confirm the details transfered or created for the agreement are what the user wants.
+    put:
+      deprecated: false
+      description: Replace the name transferred or created for the procurement automatically with what the user wants.
       tags:
         - Create Procurement Project
       operationId: ConfirmProcurementProject
       requestBody:
         content:
           application/vnd.api+json:
             schema:
-              $ref: 'https://raw.githubusercontent.com/Crown-Commercial-Service/ccs-api-definitions-common/main/CCS_OCDS_Standards/CCS-OCDS_Schema.yaml#/components/schemas/Release'
+              properties:
```

Comment:
@ronin1972 Could we add the following to this inline schema def please, to aid the code generation:
```
type: object
title: ProcurementProjectName
```
Without this, we have no wrapper object to use in the generated code and it makes things surprisingly awkward (as @adrianmilne has been discovering!).  Feel free to change the title.  Same required for the event name update 👍